sdk/tools/xml2sdb/xml2sdb.h

```diff
@@ -28,138 +28,145 @@ typedef std::basic_string<WCHAR> sdbstring;
 
 struct Database;
 
+enum PlatformType
+{
+    PLATFORM_NONE = 0,
+
+    // This is all we support for now
+    PLATFORM_X86 = 1,
+    PLATFORM_AMD64 = 4,
```

Comment:
Is there a reason for the gap between x86 and amd64? (skipping 2)

Reply:
Yeah, in MS's version there is some funkiness going on with i386 that I have not entirely figured out yet.